fleet/lib/rook-ceph-conf/charts/konkong/templates/cm-cephcli.yaml

```diff
@@ -10,6 +10,34 @@ data:
     ceph orch set backend ""
     ceph mgr module disable rook
 
+    waitfornfs rsphome
+    ceph nfs export rm rsphome /rsphome
+    ceph nfs export create cephfs rsphome /rsphome rsphome
+
+    waitfornfs lsstdata
+    ceph nfs export rm lsstdata /lsstdata
+    ceph nfs export create cephfs lsstdata /lsstdata lsstdata
+
+    waitfornfs project
+    ceph nfs export rm project /project
+    ceph nfs export create cephfs project /project project
+
+    waitfornfs scratch
+    ceph nfs export rm scratch /scratch
+    ceph nfs export create cephfs scratch /scratch scratch
+
+    waitfornfs obs-env
+    ceph nfs export rm obs-env /obs-env
+    ceph nfs export create cephfs obs-env /obs-env obs-env
+
+    waitfornfs auxtel
+    ceph nfs export rm auxtel /auxtel
+    ceph nfs export create cephfs auxtel /auxtel auxtel
+
+    waitfornfs comcam
+    ceph nfs export rm comcam /comcam
+    ceph nfs export create cephfs comcam /comcam comcam
+
     ceph mgr module enable rook
     ceph orch set backend rook
```

Comment:
We want to leave the orch disabled now.  See elqui/pillan as a reference.

Reply:
left them as the one in elqui 👍🏻